### PR TITLE
Add a Travis build

### PR DESCRIPTION
**Pull request by andy-armstrong**
## Description

Add a Travis configuration to do a production build. There are no tests at this point, but the intention is just to make sure that the library itself can still be packaged.
  
## Post-review
- [ ] Squash commits into discrete sets of changes with descriptive commit messages.

## Reviewers
- [x] @bjacobel

### CODE REVIEW

**Comment by andy-armstrong** [comment]
@bjacobel It seems that the Pattern Library sass build is running out of memory on Travis. I'm not sure how best to fix this other than to try switching to Webpack... any ideas?

**Comment by andy-armstrong** [comment]
@bjacobel @dsjen @dan-f @AlasdairSwan please review this one too. I've switched over to using Webpack instead of gulp-sass primarily so that Travis can build without running out of memory (although we wanted to move to Webpack anyway). 

In order to land this, I also had to remove the precompiled CSS from the Pattern Library package, as Webpack cannot generate CSS that includes relative paths. From talking with @talbs, it doesn't seem that anyone should be using the precompiled CSS, as they should instead be importing the partials.

**Comment by bjacobel** [comment]
:thumbsup:

**Comment by bjacobel** [comment]
Just a question - why (in the `travis.yml`) does this build need Chrome and Firefox running?

**Comment by andy-armstrong** [comment]
@bjacobel Great question. That was bad cut-and-paste from the UI Toolkit Travis configuration. I've removed them and also tried @dan-f's suggested way to specify multiple languages.

File: gulp/tasks/pldoc.js [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-    handleErrors = require('../util/handleErrors'),
-    sass = require('gulp-sass'),
-    sourcemaps = require('gulp-sourcemaps'),
-    rename = require('gulp-rename'),
-    size = require('gulp-filesize'),
-    webpack = require('gulp-webpack'),
-    ghPages = require('gulp-gh-pages');
+    webpackUtil = require('../util/webpack'),
+    ghPages = require('gulp-gh-pages'),
+    runSequence     = require('run-sequence');
```
Comment (AlasdairSwan) [inline gulp/tasks/pldoc.js]: Nitpick: spacing

**Comment by AlasdairSwan** [comment]
Just a nitpick. Once tests pass 👍

**Pull request merged**